lib/puppet/provider/rabbitmq_plugin/rabbitmqplugins.rb

```diff
@@ -11,7 +11,7 @@ def self.instances
       rabbitmqplugins('list', '-E', '-m')
     end
 
-    plugin_list.split(%r{\n}).map do |line|
+    plugin_list.split(%r{\n}).grep_v(/^Listing plugins.*$/).map do |line|
```

Comment:
Do you think it would work to do something similar to this to conditionally based on the version fact?
https://github.com/voxpupuli/puppet-rabbitmq/blob/00e1cc0d9b1075b786f40fff963fb3314a7c3a82/lib/puppet/provider/rabbitmq_plugin/rabbitmqplugins.rb#L24

Reply:
I'm taking a look at this option locally (either way, I think this will need a unit test)